databricks_cli/dbfs/api.py

```diff
@@ -168,11 +241,23 @@ def delete(self, dbfs_path, recursive, headers=None):
             break
         click.echo("\rDelete finished successfully.\033[K")
 
+    @retry(wait=wait_random_exponential(multiplier=1, max=30), 
+           retry=retry_if_exception_type(RateLimitException))
     def mkdirs(self, dbfs_path, headers=None):
-        self.client.mkdirs(dbfs_path.absolute_path, headers=headers)
+        try:
+            self.client.mkdirs(dbfs_path.absolute_path, headers=headers)
+        except HTTPError:
+            click.echo("\rRate Limit Exceeded. Trying again.")
+            raise RateLimitException
```

Comment:
Why raise `RateLimitException` regardless of the `HTTPError` status code? It should be only for `429` and raise original exception otherwise.

Same for `move`, potentially in other places too.